### ISSUE TITLE
POST and debugbar

### ISSUE THREAD

**Issue by keulu** [bcit-ci/CodeIgniter4#172]
Hi Guys, 

I've some trouble with easyPHP on windows 10, nginx and PHP 7.0.8

I can't send POST value. debug bar make an error 500, if a desactivate it, 200 (OK) and i get my post

**Comment by lonnieezell** [comment]
Please check your PHP error log for details of what the error actually is. The 500 error is so generic it's impossible for us to debug that without replicating your environment. If we have that error, though, we know what to look for.

**Comment by keulu** [comment]
got it !

Sry, didn't saw your answer

```
2016/07/21 15:33:28 [error] 13#0: *1954 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Method CodeIgniter\HTTP\Header::__toString() must not throw an exception, caught ErrorException: Array to string conversion in /var/www/ci4.dev/application/Views/errors/html/error_exception.php on line 270
PHP message: PHP Fatal error:  Method CodeIgniter\HTTP\Header::__toString() must not throw an exception, caught ErrorException: Array to string conversion in /var/www/ci4.dev/application/Views/errors/html/error_exception.php on line 270" while reading response header from upstream, client: 192.168.16.9, server: ci4.dev, request: "POST /test/index HTTP/1.0", upstream: "fastcgi://192.168.16.6:9000", host: "ci4.dev", referrer: "https://ci4.dev/test/index"
```

Got nginx and php7.0.8 on debian 8

I thinked that was because windows sucks, but got the same probleme on linux :(


well, my form is pretty simple

```
<form method="post" action="">
                    <input type="hidden" value="bar" name="foo">
                    <input type="submit" value="Send" name="Send">
</form>
```

**Comment by lonnieezell** [comment]
Awesome. Very helpful thanks. I don't know if I'll have a chance to look into in the next few days as my schedule is slammed currently, but will get to it as soon as possible.

**Comment by lonnieezell** [comment]
Fixed in #308
